Ginger/GingerCore/GingerCore.csproj

```diff
@@ -885,7 +884,7 @@
       <Version>1.8019.4006.503</Version>
     </PackageReference>
     <PackageReference Include="SSH.NET">
-      <Version>2016.1.0</Version>
+      <Version>2020.0.1</Version>
```

Comment:
did you tested all possible impact due to upgrade also on Unix driver and other depended code?

Reply:
check agent , what we have more?